bin/prettier.js

```diff
@@ -193,7 +191,11 @@ if (stdin) {
       if (write) {
         // Don't write the file if it won't change in order not to invalidate
         // mtime based caches.
-        if (output !== input) {
+        if (output === input) {
+          console.log("[ignore] %s", filename);
```

Comment:
even though this has been ignored, it still took time to actually prettify that file